### ISSUE TITLE
copyFile fails silently

### ISSUE THREAD

**Issue by Otto-AA** [jeff-zucker/solid-file-client#26]
I've had some trouble implementing a rename function (copying and removing the old) and while searching for the error I noticed that `copyFile` failed silently when the original file doesn't exist. I don't know, if this also affects other scenarios.

Here is an example:
```javascript
const from = 'abc';
const to = 'xyz';
fileClient.copyFile(from, to)
    .then(success => {
        console.log(`Copied ${from} to ${to}.`);
    }, err => console.log(err));
Promise { <state>: "pending" }
// Outputs: Copied abc to xyz.
```

**Comment by jeff-zucker** [comment]
I am in the middle of a re-write of the copy function, the new one will be based on solid-recursive-copy and behave quite differently (and more powerfully).  In the meantime, I believe you can either first do a readFile() and only perform the copy if it's successful, or check the return value from copyFile() within the then() block - it should have the name of the newly created file if one was created.  Thanks for pointing this out, I 'll check to see if it impacts other things.

**Comment by jeff-zucker** [comment]
Fixed in version 1.0.0.
